### DISCUSSION TITLE
How to inspect the memory usage without device or with dummy device?

### DISCUSSION THREAD

**Discussion by BrllsRX** [qutech/qupulse#733] [Q&A]
@terrorfisch Is there a test unit to inspect memory usage for each type of AWGs? For example, at the end of [Task 2 of snake charge scan](https://github.com/BrllsRX/qupulse/blob/researchTech/doc/source/examples/99SnakeChargeScan.ipynb), I intended to use a dummyHDAWG to test a sequenced program and would like to have an inspection of the memory usage. Any easy approach to do so?

**Comment by shumpohl** [comment]
There is not way to do that I know of.
